packages/clerk-js/src/utils/user.ts

```diff
@@ -33,5 +35,12 @@ export const getIdentifier = (user: Partial<UserResource>): string => {
 };
 
 export const isDefaultProfileImage = (url: string) => {
-  return !(url || '').includes('gravatar');
+  let encoded: string;
+  try {
+    encoded = new URL(url).pathname.replace('/', '');
+  } catch {
+    return false;
+  }
+  const decoded = decodeBase16(encoded);
+  return decoded.includes('default');
```

Comment:
🔧  i think this function should be with both `profileImageUrl` and `imageUrl`!

Reply:
Correct, fixing

Reply:
Done, thanks @dimkl